src/index.html

```diff
@@ -5,8 +5,348 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home"></a>
+      <nav class="header__navigation">
+        <ul class="header__navlist">
+          <li class="header__listitem">
+            <a href="#apple" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#smartphones" class="header__link">Smartphones</a>
+          </li>
+          <li class="header__listitem">
+            <a
+            href="#l&c"
+            class="header__link"
+            data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__listitem">
+            <a href="#gadgets" class="header__link">Gadgets</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#tablets" class="header__link">Tablets</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#photo" class="header__link">Photo</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#video" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          class="card__image"
+          alt="cardimage"
```

Comment:
```suggestion
          alt="APPLE A1419 iMac 27"
```
the alt attribute should describe what is in this image